test/chrome_test.js

```diff
@@ -62,9 +64,7 @@ describe("StorageArea", () => {
       }).catch(done);
     });
 
-    it("set/remove a value in IDB", (done) => {
-      const area = new StorageArea("http://localhost:8080/v1/", "sync");
-
+    it("set/remove a value in IDB", function(done) {
       area.set({"something": 1}, function () {
         area.remove("something", function() {
           area.get("something", function(items) {
```

Comment:
Huh, callbacks. That's probably not part of this patch but have you considered using something like [promisify](https://www.npmjs.com/package/promisify) for these?